plugin/storage/grpc/shared/grpc_client.go

```diff
@@ -222,13 +222,23 @@ func (c *grpcClient) WriteSpan(ctx context.Context, span *model.Span) error {
 	_, err := c.writerClient.WriteSpan(ctx, &storage_v1.WriteSpanRequest{
 		Span: span,
 	})
+
 	if err != nil {
 		return fmt.Errorf("plugin error: %w", err)
 	}
 
 	return nil
 }
 
+func (c *grpcClient) Close() error {
+	_, err := c.writerClient.Close(context.Background(), &storage_v1.CloseWriterRequest{})
```

Comment:
nit: does it make sense to define a timeout/deadline for the context in this case?

Reply:
1) In-process writer currently has not timeout on shutdown.
2) Plugin RPC works over unix socket. Network timeout can't happen. If child process dies, we will receive error right away.
3) In my use case, there will be external timeout on instance shutdown from the deployment system.

So I think using context.Background is OK.